packages/@okta/vuepress-site/guides/sign-into-spa/section03/angular/attachcomponentsetc.md

```diff
@@ -1,23 +1,67 @@
-## Attach Components to Routes
-You need to provide these routes in your sample application so that we can sign the user in and handle the callback from Okta. We show you how to set these up below using [Angular Router](https://angular.io/guide/router):
+## Provide the Login and Logout Buttons
 
-- `/`: A default home page to handle basic control of the app.
-- `/implicit/callback`: Handle the response from Okta and store the returned tokens.
+In the relevant location in your application, you want to provide `Login` and `Logout` buttons for the user. The `OktaAuthService` is injected in your component's constructor. You can use the `oktaAuth.isAuthenticated()` method to show/hide the correct button. For example:
```

Comment:
Also - this section? The content I added was from the existing Angular Quick Start guide. Am assuming that things have changed/need to change with this content since it was created? I'm no expert for sure...but always want to double check things.